diaries.html

```diff
@@ -384,9 +388,11 @@ <h1>Freshbits Diaries</h1>
     <section>
         <div class="container">
             <div class="freshbits-diaries">
-                <div class="freshbits-diaries-row">
+                <div class="freshbits-diaries-row" id="fun-blast">

```

Comment:
Add month and year in all the entries so that we do not face the duplicate issue

`fun-blast-feb-2022`